src/commands/report.ts

```
@@ -394,4 +335,129 @@ async function generateHTMLReport(data: ReportData, templatePath?: string): Prom
+  }
+}
+
+async function generateReportOutput(reportData: ReportData, options: ReportOptions): Promise<string> {
+  try {
+    switch (options.format) {
+      case 'json':
+        return JSON.stringify(reportData, null, 2);
+      case 'html':
+        return await generateHTMLReport(reportData, options.template);
+      case 'markdown':
+      default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)